charts/loki-distributed/templates/_helpers.tpl

```diff
@@ -145,11 +145,11 @@ Return the appropriate apiVersion for PodDisruptionBudget.
 Return the appropriate apiVersion for HorizontalPodAutoscaler.
 */}}
 {{- define "loki.hpa.apiVersion" -}}
-  {{- if and (.Capabilities.APIVersions.Has "autoscaling/v2") (semverCompare ">=1.23-0" .Capabilities.KubeVersion.Version) -}}
-    {{- print "autoscaling/v2" -}}
-  {{- else -}}
-    {{- print "autoscaling/v2beta1" -}}
-  {{- end -}}
+  {{- if .Capabilities.APIVersions.Has "autoscaling/v2" }}
+    {{- print "autoscaling/v2" }}
+  {{- else }}
```

Comment:
Should we check for `autoscaling/v2beta2` and `autoscaling/v2beta1` and set those when supported also.  I see `autoscaling/v2beta1` is in 1.24, which doesn't seem that long ago, even if most of us are running newer versions.